### ISSUE TITLE
Not up-to-date version of vmauth binary in docker image

### ISSUE THREAD

**Issue by ofen** [VictoriaMetrics/VictoriaMetrics#677]
**Describe the bug**
According to [**wiki**](https://victoriametrics.github.io/vmauth.html#security) vmauth should have -tls* flags but they are missing in recent docker image **v1.39.3-cluster**

**To Reproduce**
```bash
$ docker run victoriametrics/vmauth:v1.39.3-cluster -tls
flag provided but not defined: -tls
...
```

**Expected behavior**
tls* flags available

**Screenshots**
[![Screenshot-from-2020-08-06-23-10-17.png](https://i.postimg.cc/q76XVbsh/Screenshot-from-2020-08-06-23-10-17.png)](https://postimg.cc/cKZ8fc50)

**Version**
```bash
$ docker run victoriametrics/vmauth:v1.39.3-cluster -version
vmauth-20200805-101220-tags-v1.39.3-cluster-0-g76064ba9e
```

**Comment by tenmozes** [comment]
the version from master branch contains tls 
``` bash
docker run --rm victoriametrics/vmauth:v1.39.3 --help

 -tls
    	Whether to enable TLS (aka HTTPS) for incoming requests. -tlsCertFile and -tlsKeyFile must be set if -tls is set
  -tlsCertFile string
    	Path to file with TLS certificate. Used only if -tls is set. Prefer ECDSA certs instead of RSA certs, since RSA certs are slow
  -tlsKeyFile string
    	Path to file with TLS key. Used only if -tls is set
```

**Comment by valyala** [comment]
The issue should be fixed in the commit 67cacb22ac5ddad6a2247323f78b85748251115e . This commit will be included in the next release of VictoriaMetrics.

**Comment by valyala** [comment]
The bugfix that adds `-tls*` command-line flags to `*-cluster` vmauth has been included in [v1.39.4](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
